ORGANISATIONS.yaml

```diff
@@ -45,6 +45,13 @@ deNBI:
     name: de.NBI
     url: https://www.denbi.de/
 
+deKCD:
+    name: German Competence Center Cloud Technologies for Data Management and Processing (de.KCD) 
+    short_name: "de.KCD"
+    joined: 2025-03
+    github: https://github.com/deKCD
```

Comment:
this will be automatically constructed, key is only used to disable that (e.g. `github: false`

```suggestion
```